### ISSUE TITLE
lr_finder documentation

### ISSUE THREAD

**Issue by BramVanroy** [PyTorchLightning/pytorch-lightning#11077]
## 📚 Documentation

It seems to me that [the documentation of the lr_finder](https://pytorch-lightning.readthedocs.io/en/stable/advanced/lr_finder.html) is ambiguous. It states

> It is recommended to not pick the learning rate that achieves the lowest loss, but instead something in the middle of the sharpest downward slope (red point).

However, in the image the red dot is not on the sharpest slope. The sharpest slope is between 10^-2 and 10^-1. 

So either this is a mistake (or ambiguity) in the documentation, or the implementation picks the lr based on the wrong slope. (In my own work I have found that the lr_finder underestimates the best lr value.)

cc @borda

**Comment by rohitgr7** [comment]
it specifies the **sharpest downward slope**.. so looks correct to me.

> The sharpest slope is between 10^-2 and 10^-1.

this is the sharpest upward slope.

**Comment by BramVanroy** [comment]
I'm stupid. :-)
